com.unity.sg2/Editor/GraphUI/DataModel/GraphDataNodeModel.cs

```diff
@@ -265,31 +266,18 @@ void initCallback(IConstant e)
                     if (e == null)
                         return;
                     var shaderGraphModel = ((ShaderGraphModel)GraphModel);
-                    var handler = shaderGraphModel.GraphHandler;
-                    try
-                    {
-                        var possiblyNodeReader = handler.GetNode(nodeId);
-                    }
-                    catch
-                    {
-                        handler = shaderGraphModel.RegistryInstance.DefaultTopologies;
-                    }
-                    // don't do this, we should have a fixed way of pathing into a port's type information as opposed to its header/port data.
-                    // For now, we'll fail to find the property, fall back to the port's body, which will parse it's subfields and populate constants appropriately.
-                    // Not sure how that's going to work for data that's from a connection!
```

Comment:
We don't use the handler here to initialize anymore. Note that we want to ask GTF to initialize constants with their owner so we don't need the initialization callback at all.